src/main/resources/user-create-mapping.conf

```diff
@@ -0,0 +1,416 @@
+sunbird_user_teacher_id_provider = "shaalakosh"
+teacher {
```

Comment:
Add comments to explain format so that an adopter can do required configuration changes.